sdcm/scylla_bench_thread.py

```diff
@@ -175,7 +175,7 @@ def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
 
         log_file_name = os.path.join(node.logdir, f'scylla-bench-l{loader_idx}-{uuid.uuid4()}.log')
         # Select first seed node to send the scylla-bench cmds
-        ips = node_list[0].cql_ip_address
+        ips = ",".join([n.cql_ip_address for n in node_list])
```

Comment:
@fruch did you try to run? AFAIR scylla-bench is not able to run on few nodes. But maybe it was changed?

Reply:
seem like it's working just fine

Reply:
```
Command "/$HOME/go/bin/scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=1     -partition-offset=476 -clustering-row-size=uniform:8192..10240  -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -iterations 10 -error-at-row-limit 1000 -nodes 10.0.1.204,10.0.1.72,10.0.2.215,10.0.1.70,10.0.2.102" finished with status 0
```